config/kvconfig.go

```diff
@@ -0,0 +1,235 @@
+// Code generated by go-bindata.
```

Comment:
Does this file need to build committed?  From what I understand it gets autogenerated when `make build` and `make release` are called.

Reply:
Yeah, unfortunately since the config.go file calls `Asset` which exists in this file.